app/code/Magento/Cron/Observer/ProcessCronQueueObserver.php

```diff
@@ -298,7 +298,7 @@ protected function _runJob($scheduledTime, $currentTime, $jobConfig, $schedule,
 
         if (!isset($jobConfig['instance'], $jobConfig['method'])) {
             $schedule->setStatus(Schedule::STATUS_ERROR);
-            throw new \Exception('No callbacks found');
+            throw new \Exception('No callbacks found for cron job %s', $jobCode);
```

Comment:
Did you miss `sprintf` call here? Better use `"This is $string"`.

Also, please provide more detailed steps to reproduce.

Reply:
@orlangur re pushed with sprintf added